### ISSUE TITLE
Quirk suggestion: La-Z-Boy

### ISSUE THREAD

**Issue by carlylemiii** [apple/password-manager-resources#435]
Hoping we can save others future pain, because this site was awful signing up on. 😬 

https://www.la-z-boy.com/modals/create-account.jsp

1. This site uses a maxlength attribute of 16:

<img width="882" alt="CleanShot 2021-03-13 at 00 00 14@2x" src="https://user-images.githubusercontent.com/29067983/111021164-1b363a00-8390-11eb-94f2-df3f24086b36.png">

2. But its actual maxlength is 15…

<img width="552" alt="CleanShot 2021-03-12 at 23 59 56@2x" src="https://user-images.githubusercontent.com/29067983/111021173-28532900-8390-11eb-8fa1-f7ec76f69058.png">

3. But if you go to the change-password page (https://www.la-z-boy.com/account/profile.jsp), they claim their min is 5…

<img width="432" alt="CleanShot 2021-03-13 at 00 04 17@2x" src="https://user-images.githubusercontent.com/29067983/111021190-4456ca80-8390-11eb-9bde-5a9bb4b0392a.png">

4. But if you try a 5-character password, you're told it needs to be 6…

<img width="655" alt="CleanShot 2021-03-13 at 00 12 06@2x" src="https://user-images.githubusercontent.com/29067983/111021285-d1018880-8390-11eb-9879-1d3f74d2c8f7.png">

5. They don't tell you they don't accept special characters until after you've failed that check (on the sign-up page, that is):

<img width="566" alt="CleanShot 2021-03-12 at 23 59 41@2x" src="https://user-images.githubusercontent.com/29067983/111021217-7d8f3a80-8390-11eb-940c-ce7989b52ff0.png">

**tl;dr**: 6–15 characters, only letters and numbers allowed
